plugins/presentation/src/plugin.ts

```diff
@@ -91,11 +91,11 @@ export default (platform: Platform, deps: { core: CoreService, i18n: I18n }): Pr
         const uxAttribute = model.as(attribute, ui.mixin.UXAttribute)
 
         const label = await i18nService.translate(uxAttribute.label)
-        const placeholder = uxAttribute.placeholder ? await i18nService.translate(uxAttribute.placeholder) : label
+        const placeholder = (uxAttribute.placeholder != null) ? await i18nService.translate(uxAttribute.placeholder) : label
```

Comment:
seems to be `undefined` as well as presenter below

Reply:
Fixed. thanks.